src/05-derivatives/01-introduction.md

```diff
@@ -0,0 +1,315 @@
+| GeneratedBy           | REQUIRED. List of JSON objects with at least one element.                                                                                                                |
+| SourceDatasets        | RECOMMENDED. A list of objects specifying the locations and relevant attributes of all source datasets. Valid fields in each object include `URL`, `DOI`, and `Version`. |
+
+Each object in the `GeneratedBy` list includes the following REQUIRED, RECOMMENDED
+and OPTIONAL keys:
+
+| **Key name** | **Description**                                                                                                                                                                                              |
+| ------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| Name         | REQUIRED. Name of the pipeline or process that generated the outputs. Use `"Manual"` to indicate the derivatives were generated by hand, or adjusted manually after an initial run of an automated pipeline. |
+| Version      | RECOMMENDED. Version of the pipeline.                                                                                                                                                                        |
+| Description  | OPTIONAL. Plain-text description of the pipeline or process that generated the outputs. RECOMMENDED if `Name` is `"Manual"`.                                                                                 |
+| CodeURL      | OPTIONAL. URL where the code used to generate the derivatives may be found.                                                                                                                                  |
+| Container    | OPTIONAL. Object specifying the location and relevant attributes of software container image used to produce the derivative. Valid fields in this object include `Type`, `Tag` and `URI`.                    |
+
+If a derived dataset is stored as a subfolder of the raw dataset, then the `Name` field
+of the first `GeneratedBy` object MUST be a substring of the derived dataset folder name.
+That is, in a directory `<dataset>/derivatives/<pipeline>[-<variant>]/`, the first
+`GeneratedBy` object should have a `Name` of `<pipeline>`.
+
+Example:
+
+```JSON
+{
+  "Name": "FMRIPREP Outputs",
+  "BIDSVersion": "1.4.0",
+  "GeneratedBy": [
+    {
+      "Name": "FMRIPREP",
+      "Version": "1.2.5",
+      "Container": {
+        "Type": "docker",
+        "Tag": "poldracklab/fmriprep:1.2.5"
+        }
+    },
+    {
+      "Name": "Manual",
+      "Description": "Re-added RepetitionTime metadata to bold.json files"
+    }
+  ],
```

Comment:
Please review the new `GeneratedBy` field, which replaced the `PipelineDescription` field. This field is mandatorily a list, and I have added a keyword "Manual" to indicate manual modification and a free-text "Description" field so a curator can describe their adjustments.

@emdupre Could you check whether the pipeline name/folder name matching text still makes sense to you?

Reply:
I missed the one week comment period on this, it sounds like -- sorry about that ! I think the current phrasing is less clear but I can old comments as a future PR ?